### PR TITLE
added more edge cases to tests

### PR DESCRIPTION
**Pull request by wohlfea**

### CODE REVIEW

File: src/test_queue.py [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
 DEQUEUE = [([], IndexError),
-           (['one', 'two', 'three', 'four'], 'one')]
+           (['one', 'two', 'three', 'four'], 'one'),
+           (['one'], 'one'),
+           (['Uno'], 'Uno')]
 ENQUEUE = [([], None),
            (['one'], 'one'),
-           (['one', 'two', 'three'], 'three')]
+           (['one', 'two', 'three'], 'three'),
+           (['How', 'are', 'you', 'doing!?'], 'doing!?')]
```
Comment (joelstanner) [inline src/test_queue.py]: Better tests. Good.
